scripts/update_lock_files.sh

```diff
@@ -19,7 +19,7 @@ else
 fi
 
 # Array of submodules
-submodules=("app" "endorser" "trustregistry" "webhooks")
```

Comment:
I think it'd be better to move away from `scripts/` and standardize on using [Mise Tasks](https://mise.jdx.dev/tasks/).

Reply:
Agreed :+1: